packages/core-mobile/app/temp/caip2ChainIds.ts

```diff
@@ -42,13 +44,15 @@ enum BlockchainId {
   P_CHAIN = `${BlockchainNamespace.AVAX}:11111111111111111111111111111111LpoYY`,
   P_CHAIN_TESTNET = `${BlockchainNamespace.AVAX}:fuji-11111111111111111111111111111111LpoYY`,
   X_CHAIN = `${BlockchainNamespace.AVAX}:2oYMBNV4eNHyqk2fjjV5nVQLDbtmNJzq5s3qs3Lo6ftnC6FByM`,
-  X_CHAIN_TESTNET = `${BlockchainNamespace.AVAX}:2JVSBoinj9C2J33VntvzYtVJNZdN2NKiwwKjcumHUWEb5DbBrm`
+  X_CHAIN_TESTNET = `${BlockchainNamespace.AVAX}:2JVSBoinj9C2J33VntvzYtVJNZdN2NKiwwKjcumHUWEb5DbBrm`,
+  P_CHAIN_DEVNET = `${BlockchainNamespace.AVAX}:custom11111111111111111111111111111111LpoYY`
 }
 
 export const isPChainId = (caip2ChainId: string): boolean => {
   return (
     caip2ChainId === AvalancheCaip2ChainId.P ||
-    caip2ChainId === AvalancheCaip2ChainId.P_TESTNET
+    caip2ChainId === AvalancheCaip2ChainId.P_TESTNET ||
+    caip2ChainId === 'avax:N5PeRoKwDDREF0awn2GPsb9sHnGRzdEX'
```

Comment:
- [ ] remove this chainId after Etna activation

Reply:
https://ava-labs.atlassian.net/browse/CP-9539